build-helpers/tap-driver.sh

```diff
@@ -370,7 +370,7 @@ function setup_result_obj(line)
   sub("^(not )?ok[ \t]*", "", line)
 
   # If the result has an explicit number, get it and strip it; otherwise,
-  # automatically assing the next progresive number to it.
+  # automatically assign the next progressive number to it.
```

Comment:
I'm not particularly confident about `assign`, it's possible it should be `passing`